docs/rpc/grpc.md

```diff
@@ -19,35 +19,13 @@ Below is a table of attributes that SHOULD be included on client and server gRPC
 <!-- semconv rpc.grpc -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `rpc.grpc.request.metadata.<key>` | string[] | gRPC request metadata, `<key>` being the normalized gRPC Metadata key (lowercase, with `-` characters replaced by `_`), the value being the metadata values. [1] | `rpc.grpc.request.metadata.my_custom_metadata_attribute=["1.2.3.4", "1.2.3.5"]` | Opt-In |
-| `rpc.grpc.response.metadata.<key>` | string[] | gRPC response metadata, `<key>` being the normalized gRPC Metadata key (lowercase, with `-` characters replaced by `_`), the value being the metadata values. [2] | `rpc.grpc.response.metadata.my_custom_metadata_attribute=["attribute_value"]` | Opt-In |
-| `rpc.grpc.status_code` | int | The [numeric status code](https://github.com/grpc/grpc/blob/v1.33.2/doc/statuscodes.md) of the gRPC request. | `0` | Required |
+| [`rpc.grpc.request.metadata.<key>`](../attributes-registry/rpc.md) | string[] | gRPC request metadata, `<key>` being the normalized gRPC Metadata key (lowercase, with `-` characters replaced by `_`), the value being the metadata values. [1] | `rpc.grpc.request.metadata.my_custom_metadata_attribute=["1.2.3.4", "1.2.3.5"]` | Opt-In |
+| [`rpc.grpc.response.metadata.<key>`](../attributes-registry/rpc.md) | string[] | gRPC response metadata, `<key>` being the normalized gRPC Metadata key (lowercase, with `-` characters replaced by `_`), the value being the metadata values. [2] | `rpc.grpc.response.metadata.my_custom_metadata_attribute=["attribute_value"]` | Opt-In |
+| [`rpc.grpc.status_code`](../attributes-registry/rpc.md) | int | The [numeric status code](https://github.com/grpc/grpc/blob/v1.33.2/doc/statuscodes.md) of the gRPC request. | `0` | Required |
 
 **[1]:** Instrumentations SHOULD require an explicit configuration of which metadata values are to be captured. Including all request metadata values can be a security risk - explicit configuration helps avoid leaking sensitive information.
 
 **[2]:** Instrumentations SHOULD require an explicit configuration of which metadata values are to be captured. Including all response metadata values can be a security risk - explicit configuration helps avoid leaking sensitive information.
-
-`rpc.grpc.status_code` MUST be one of the following:
```

Comment:
Also here, probably the table needs to be rendered with `full` to show the enum values

Reply:
Those are available in the registry, what is the main guidance in such cases - do we try to make explicit all the values or rely on the links
```
[`rpc.grpc.status_code`](../attributes-registry/rpc.md)
```

Reply:
@joaopgrassi I agree with you, the change should not change the table content-wise. The problem with `full` is that it will also render all the inherited attributes. I think we are missing a rendering option in the build-tools that would allow to only render all the explicitly listed attributes but with all their details / constraints. 

Or I'm just not aware of such an option.
@lmolkova Do you know of a way doing so? Otherwise I'd extend the build-tools with such an option.

Reply:
I think I found a way of doing so, by combining `full` and `tag`

Reply:
@joaopgrassi The rendering should be fixed now. PTAL

Reply:
Tbh I don't really like using this inherit property. I'm not sure it's useful and it seems it causes more damage than any good. Like we have seen in the db conventions, it was used there but not "full" so we don't know the state. I think it's always best to list them all.